.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+Closes #issue-number-here (replace with issue number)
+Description
```

Comment:
I think you lost some Markdown syntax while copy/pasting it, like `#` for Headers and `- [ ]` for the TODO list below.

Reply:
I've updated it, do let me know if any more changes would be appropriate.